src/gui/activitydata.h

```diff
@@ -57,6 +57,7 @@ class Activity
     QString _message;
     QString _file;
     QUrl _link;
+    QString _linkText;
```

Comment:
Maybe document which field are html escaped.

Personally, i'd keep all these string unescaped.  and escape them right before putting them in some html text.